bundles/plugins/org.bonitasoft.studio.common/src/org/bonitasoft/studio/common/emf/tools/ModelHelper.java

```diff
@@ -730,6 +468,26 @@ public static DataType getDataTypeForID(final EObject elem, final String name) {
         return null;
     }
 
+    public static DataType getDataTypeByClassName(final MainProcess dataTypeContainer, final String returnTypeClassname) {
+        checkArgument(dataTypeContainer != null);
+        checkArgument(returnTypeClassname != null);
+        if (returnTypeClassname.equals(Boolean.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(BooleanType.class), null);
+        } else if (returnTypeClassname.equals(String.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(StringType.class), null);
+        } else if (returnTypeClassname.equals(Double.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(DoubleType.class), null);
+        } else if (returnTypeClassname.equals(Long.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(LongType.class), null);
+        } else if (returnTypeClassname.equals(Integer.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(IntegerType.class), null);
+        } else if (returnTypeClassname.equals(Date.class.getName())) {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(DateType.class), null);
+        } else {
+            return find(dataTypeContainer.getDatatypes(), instanceOf(JavaType.class), null);
+        }
+    }
+
```

Comment:
it reminds me code in org.bonitasoft.studio.data.util.DataUtil.getTechnicalTypeFor(AbstractProcess, String). Possible to factorize?